src/content/quickstart/kind/_index.md

```diff
@@ -61,12 +61,7 @@ To manually verify the deployment follow the steps below.
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 create deployment nginx --image=nginx
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 expose deployment nginx --port=80
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 -n default apply -f - <<EOF
-apiVersion: lighthouse.submariner.io/v2alpha1
-kind: ServiceExport
-metadata:
-  name: nginx
-EOF
-kubectl --kubeconfig output/kubeconfigs/kind-config-cluster2 -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash curl nginx.default.svc.supercluster.local:8080
+subctl export service --namespace default nginx
 ```
 
 {{% notice info %}}
```

Comment:
It sees like we don't need this note anymore since we now instruct the user to use subctl which takes care of this.

Reply:
Done.

Reply:
It wasn't removed.

Reply:
oh forgot git add . Fixed it